parsers/test/test_PE.py

```diff
@@ -0,0 +1,78 @@
+from urllib.parse import urlencode
+
+from freezegun import freeze_time
+from requests import Session
+from requests_mock import POST, Adapter
+from snapshottest import TestCase
+
+from electricitymap.contrib.lib.types import ZoneKey
+from parsers.PE import API_ENDPOINT, fetch_production
+
+
+class TestFetchProduction(TestCase):
+    def setUp(self):
+        self.adapter = Adapter()
+        self.session = Session()
+        self.session.mount("https://", self.adapter)
+        mock_file_today = open("parsers/test/mocks/PE/response_20240206.json", "rb")
+        mock_file_yesterday = open("parsers/test/mocks/PE/response_20240205.json", "rb")
+        self.adapter.register_uri(
+            POST,
+            API_ENDPOINT,
+            response_list=[
+                {"content": mock_file_today.read()},
+                {"content": mock_file_yesterday.read()},
+            ],
+        )
+
+    def test_snapshot(self):
+        production = fetch_production(
+            zone_key=ZoneKey("PE"),
+            session=self.session,
+        )
+
+        self.assertMatchSnapshot(
+            [
+                {
+                    "datetime": element["datetime"].isoformat(),
+                    "production": element["production"],
+                    "source": element["source"],
+                    "zoneKey": element["zoneKey"],
+                    "sourceType": "measured",
```

Comment:
```suggestion
                    "sourceType": element["sourceType"].value,
```
This just brings it in line with our other tests and makes it easy to update if we ever change this.